internal/node/launcher.sh

```diff
@@ -49,7 +77,9 @@ source "${RUNFILES_DIR:-/dev/null}/$f" 2>/dev/null || \
 # Case 6a is handled like case 3.
 if [[ -n "${RUNFILES_MANIFEST_ONLY:-}" ]]; then
   # Windows only has a manifest file instead of symlinks.
-  RUNFILES=${RUNFILES_MANIFEST_FILE%/MANIFEST}
+  # Normalizing the path and case insensitive removing the `/MANIFEST` part of the path
+  NORMALIZED_RUNFILES_MANIFEST_FILE_PATH=$(normalizeWindowsPath $RUNFILES_MANIFEST_FILE)
+  RUNFILES=$(sed 's|\/MANIFEST$||i' <<< $NORMALIZED_RUNFILES_MANIFEST_FILE_PATH)
```

Comment:
can we avoid depending on `sed`? do we know where that comes from on a windows machine? (does every bazel user have it?)

Reply:
Good point. I know `sed` is not POSIX compliant but in that use case I really think its okay to use it. In order to get all of this to work looks like we rely on `Git bash` to be installed so we can run the bash scripts inside of it when running the generated `bat` executable. As this is the case the GNU `sed` will be installed and we can be sure it will work as expected.

Reply:
@gregmagolan are you okay with this change?

Reply:
hmmm. can we limit the code path changes to Windows here to limit the potential blast radius?
```
RUNFILES=${RUNFILES_MANIFEST_FILE%/MANIFEST}
```
changes to 
```
  NORMALIZED_RUNFILES_MANIFEST_FILE_PATH=$(normalizeWindowsPath $RUNFILES_MANIFEST_FILE)
  RUNFILES=$(sed 's|\/MANIFEST$||i' <<< $NORMALIZED_RUNFILES_MANIFEST_FILE_PATH)
```
in linux and osx... and `normalizeWindowsPath` being called on all platforms is a bit confusing as you have to read that code to know it is a no-op on non-windows platforms.

Something like
```
if windows
    RUNFILES=$(normalizeWindowsRunfilesPath)
else
    RUNFILES=${RUNFILES_MANIFEST_FILE%/MANIFEST}
fi
```
would be clearer to read and would change the code path on Windows only.

As for using `sed` on Windows, your reasoning sounds ok given that we're already relying on git bash to run this bash script.

Reply:
It might also good to make this an opt-in via a config flag if you think there is risk here; tho from the looks of it it seems pretty safe

Reply:
Thanks for the review @gregmagolan ! I've applied the refactors you've suggested

Reply:
Looks good. Thanks for updating